src/components/Authentication/LoginPage.scss

```diff
@@ -0,0 +1,127 @@
+.logo {
+    padding-top: 1%;
+    padding-left: 1.5%;
+    .title {
+        position: absolute;
+        font-family: "Roboto Slab", serif;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 24px;
+        line-height: 26px;
+        text-decoration: none;
+        color: white;
+    }
+}
+.logo2 {
+    padding-top: 1%;
+    padding-left: 1.5%;
+    .title {
+        position: absolute;
+        font-family: "Roboto Slab", serif;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 24px;
+        line-height: 26px;
+        text-decoration: none;
+        color: #43B5D9;
+    }
+}
+
+.Login {
+    padding-top: 12%;
+    .inputRounded .MuiOutlinedInput-root {
+        border-radius: 50px;
+    }
+    .TextField fieldset {
+        border-radius: 300px;
+    }
+    .TextField {
+        width: 80%;
+        padding-bottom: 3%;
+    }
+    .forgotPassword {
+        text-decoration: none;
+        font-family: Roboto;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 15px;
+        line-height: 23px;
+        color: black;
+        padding-top: 30px;
+    }
+    .welcomeText {
+        font-family: "Roboto Slab";
+        font-style: normal;
+        font-weight: bold;
+        font-size: 52px;
+        line-height: 69px;
+        color: #43b5d9;
+        padding-bottom: 3%;
+    }
+    .optionsContainer {
+        .buttonSpacing {
+            padding-top: 3%;
+            .signInButton {
+                width: 80%;
+                background: #43b5d9;
+                border: 1px solid #43b5d9;
+                box-sizing: border-box;
+                box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.25);
+                border-radius: 50px;
+                padding: 8px 0 8px 0;
+                color: white;
+            }
+        }
+    }
+    .buttonContainer {
+        padding-top: 3%;
+        .signInButton {
+            width: 80%;
+            background: #ffffff;
+            border: 1px solid #747d88;
+            box-sizing: border-box;
+            box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.25);
+            border-radius: 50px;
+            padding: 8px 0 8px 0;
+            color: #666666;
+        }
+        .createAccountButton {
+            width: 80%;
+            background: #43b5d9;
+            border: 1px solid #43b5d9;
+            box-sizing: border-box;
+            box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.25);
+            border-radius: 50px;
+            padding: 8px 0 8px 0;
+            color: white;
+        }
+    }
+}
+.ellipse1 {
+    position: absolute;
+    width: 1941.01px;
+    height: 1304.46px;
+    left: -776.74px;
+    top: -635.8px;
+    transform: matrix(0.98, 0.19, -0.19, 0.98, 0, 0);
+}
+.ellipse2 {
```

Comment:
looks like your `.ellipse2 .picture1 .ellipse3 .ellipse4` all share `position: absolute;`

If you want multiple CSS classes to share the same styling do this:
```
.ellipse2, .picture1, .ellipse3, .ellipse4 {
    position: absolute;
}
```

[Multiple Class/ ID Reference](https://css-tricks.com/multiple-class-id-selectors/)